test/parallel/test-buffer-readuintle.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readUIntLE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readUIntLE(2);
+
+assert.strictEqual(result, 168);
+
+assert.throws(
+      () => { buf.readUIntLE(5); }, RangeError);
```

Comment:
Or, if you prefer to split it up for readability, please indent by only two lines and put the closing parenthesis on a separate line:

```js
assert.throws(
  () => { buf.readIntLE(5); }, RangeError
);
```

The indent-by-two-chars is something that should be caught by the linter but currently is not. I think it will start to be reported when we update our ESLint version. But there are a few obstacles to that at the moment.